src/collective/easyform/interfaces/easyform.py

```diff
@@ -98,19 +98,22 @@ class IEasyForm(Schema):
         defaultFactory=default_resetLabel,
         required=False,
     )
+    directives.write_permission(method=config.EDIT_ADVANCED_PERMISSION)
     method = zope.schema.Choice(
         title=_(u"label_method", default=u"Form method"),
         description=_(u"help_method", default=u""),
         default=u"post",
         required=False,
         vocabulary="easyform.FormMethods",
     )
+    directives.write_permission(form_tabbing=config.EDIT_ADVANCED_PERMISSION)
```

Comment:
IMO if someone adds fieldsets he/she wants tabbing. Otherwise no fieldsets. Opinions?

Reply:
Not really, but I would not touch it :)

Reply:
Maybe also into "Advanced" tab